amxmodx/srvcmd.cpp

```diff
@@ -63,6 +63,40 @@ void amx_command()
 
 		print_srvconsole("%d plugins, %d running\n", plugins, running);
 	}
+	else if (!strcmp(cmd, "plugin"))
+	{
+		auto id = atoi(CMD_ARGV(2));
```

Comment:
Why get id before checking argument count? Are you sure this code is valid if id isn't supplied?